static/css/section/_server.scss

```diff
@@ -1,213 +1,116 @@
-  overflow: hidden;
-}
-
-body.server-services .row-ubuntu-advantage {
-  padding-top: 30px;
+  .row-juju-maas div {
+    padding-bottom: 0;
+  }
 }
 
-body.server-services .row-enterprise .quote-canonical {
-  margin-right: 20px;
-  padding: 20px 40px 0 80px;
-}
+// @subsection /server/management
+.server-management {
 
-body.server-services .row-methodology {
-  background: #e2d4dc;
-  padding-top: $gutter-width;
-}
+  .row-community {
+    @media only screen and (min-width : $breakpoint-medium) {
+      border-bottom: 0;
+      margin-bottom: 0;
+      padding: 40px 20px;
 
-body.server-services .row-methodology .arrow-left {
-  bottom: auto;
-  top: 20px;
+      .nine-col {
+        background: #fff;
+      }
+    }
+  }
 }
 
-body.server-hyperscale h1 {
-  float: none;
-  margin-bottom: 18px;
-}
+// @subsection /server/hyperscale
+.server-hyperscale {
 
-body.server-contact-us .row-hero h2,
-body.server-thank-you .row-hero h2 {
-  color: #333;
-  padding-top: 0;
-}
+  h1 {
+    float: none;
+    margin-bottom: 18px;
+  }
 
-body.server-hyperscale {
   .partner-logo {
-    margin-bottom: 10px;
-  }
-}
+    margin: 0 auto 10px;
 
-/* Medium / Tablet viewport
--------------------------------------------------------------- */
-@media only screen and (min-width : 769px) {
-    body.server {
-        .box-padded-feature {
-            .six-col {
-              border-right: 1px dotted $warm-grey;
-            }
-
-            .six-col.last-col {
-              border-right: 0;
-            }
-
-            .border-bottom {
-              border-bottom: 1px dotted $warm-grey;
-              padding-bottom: 10px;
-              margin-bottom: 10px;
-            }
-        }
-    }
-    body.server-hyperscale {
-      .partner-logo {
-        height: 105px;
-      }
+    @media only screen and (min-width : $breakpoint-medium) {
+      height: 105px;
 
-      .partner-logo img {
+      img {
         position: relative;
         top: 50%;
         -webkit-transform: translateY(-50%);
         -ms-transform: translateY(-50%);
         transform: translateY(-50%);
       }
+    }
+  }
 
-      .welcome-hyperscale {
-        background-image: url("#{$asset-server}ab812f69-image-server-hyperscale.svg");
-        background-repeat: no-repeat;
-        background-position: 575px center;
-      }
+  .welcome-hyperscale {
+    @media only screen and (min-width : $breakpoint-medium) {
+      background-image: url("#{$asset-server}ab812f69-image-server-hyperscale.svg");
+      background-position: 575px center;
+      background-repeat: no-repeat;
     }
-    html.no-svg,
-    html.opera-mini {
-        body.server-hyperscale .welcome-hyperscale {
-          background-image: url("#{$asset-server}b47ff24b-image-server-hyperscale.png");
-        }
+  }
+}
+
+// @subsection /server/contact-us
+.server-contact-us {
+
+  .row-hero {
+    h2 {
+      color: $cool-grey;
+      padding-top: 0;
     }
+  }
+}
+
+// @fix for opera and no-svg
+.no-svg,
```

Comment:
This could instead be written inside the `.welcome-hyperscale {}` block above as:

``` sass
  .welcome-hyperscale {
    .no-svg .opera-mini & {
      @media ... {}
    }
  }
```

Which would keep all `.welcome-hyperscale` rules grouped together. Would this be more consistent with our attitude towards `@media` queries and the like?